README.md

```diff
@@ -6,6 +6,8 @@
 <!-- START doctoc generated TOC please keep comment here to allow auto update -->
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 <!-- **Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)* -->
+# DEPRECATED -- This repo is deprecated. If you want to integrate your policy engine with governance policy framework, please read https://github.com/open-cluster-management/governance-policy-framework
```

Comment:
Instead of of the H1 tag (#), i would only bold the statement. 

I suggest the following update:

**Deprecated -- This repository is deprecated. If you want to integrate your policy engine with the governance policy framework, see the `governance-policy-framework` repository.**

Reply:
done